source/documentation/administration_guide/topics/Syntax_for_the_engine-backup_Command.adoc

```diff
@@ -13,29 +13,29 @@ The `engine-backup` command works in one of two basic modes:
 
 *Basic Options*
 
-*`--mode`*:: Specifies whether the command will perform a backup operation or a restore operation. Two options are available - `backup`, and `restore`. This is a required parameter.
+*`--mode`*:: Specifies whether the command will perform a backup operation or a restore operation. Two options are available - `backup` (set by default), and `restore`. This is a required option in restore mode.
 
-*`--file`*:: Specifies the path and name of a file into which backups are to be taken in backup mode, and the path and name of a file from which to read backup data in restore mode. This is a required parameter in both backup mode and restore mode.
+*`--file`*:: Specifies the path and name of a file into which backups are to be taken in backup mode, and the path and name of a file from which to read backup data in restore mode. Defined by default as `/var/lib/ovirt-engine-backup/`.
 
-*`--log`*:: Specifies the path and name of a file into which logs of the backup or restore operation are to be written. This parameter is required in both backup mode and restore mode.
+*`--log`*:: Specifies the path and name of a file into which logs of the backup or restore operation are to be written. Defined by default as `/var/log/ovirt-engine-backup/` .
 
 *`--scope`*:: Specifies the scope of the backup or restore operation. There are four options: `all`, which backs up or restores all databases and configuration data; `files`, which backs up or restores only files on the system; `db`, which backs up or restores only the {engine-name} database; and `dwhdb`, which backs up or restores only the Data Warehouse database. The default scope is `all`.
 +
-The `--scope` parameter can be specified multiple times in the same `engine-backup` command.
+The `--scope` option can be specified multiple times in the same `engine-backup` command.
 
 *{engine-name} Database Options*
 
 The following options are only available when using the `engine-backup` command in `restore` mode. The option syntax below applies to restoring the {engine-name} database. The same options exist for restoring the Data Warehouse database. See `engine-backup --help` for the Data Warehouse option syntax.
 
-*`--provision-db`*:: Creates a PostgreSQL database for the {engine-name} database backup to be restored to. This is a required parameter when restoring a backup on a remote host or fresh installation that does not have a PostgreSQL database already configured.
+*`--provision-db`*:: Creates a PostgreSQL database for the {engine-name} database backup to be restored to. This is a required option when restoring a backup on a remote host or fresh installation that does not have a PostgreSQL database already configured. When this option is used in restore mode, the `--restore-permissions` option is added by default.
```

Comment:
You might want to add another option: `--provision-all-databases`, which provisions databases for all the dumps included in the archive. Now, if possible, this is the default. This means, that the minimum for restoring on a new system, if the backed up system used only defaults, is: `engine-backup --mode=restore --file=FILE`. You might want to change accordingly also `Restoring_a_Backup_to_a_Fresh_Installation.adoc`. See also https://bugzilla.redhat.com/1821018.